truffle/src/main/java/org/jruby/truffle/core/thread/ThreadManager.java

```diff
@@ -69,7 +69,8 @@ public static DynamicObject createRubyThread(RubyContext context) {
                 null,
                 null,
                 new AtomicBoolean(false),
-                0);
+                0,
+                null);
```

Comment:
If you initialize this to `nil()` then I think there is no need for `null` handling in other places.